mkinitcpio.conf

```diff
@@ -65,3 +65,9 @@ HOOKS=(base udev autodetect modconf block filesystems keyboard fsck)
 # COMPRESSION_OPTIONS
 # Additional options for the compressor
 #COMPRESSION_OPTIONS=()
+
+# MODULES_DECOMPRESS
+# Decompress kernel modules during initramfs creation.
+# Enable to speedup boot process, disable to save RAM
+# during early userspace. Switch (yes/no) default=yes
+#MODULES_DECOMPRESS=""
```

Comment:
How about showing the default value inline instead of in the accompanying comment:

```suggestion
# during early userspace. Switch (yes/no).
#MODULES_DECOMPRESS="yes"
```